diff --git a/node_modules/svelte/internal.mjs b/node_modules/svelte/internal.mjs
index 94f971c..ffcab27 100644
--- a/node_modules/svelte/internal.mjs
+++ b/node_modules/svelte/internal.mjs
@@ -119,7 +119,9 @@ function insert(target, node, anchor) {
     target.insertBefore(node, anchor || null);
 }
 function detach(node) {
-    node.parentNode.removeChild(node);
+    if (node.parentNode) {
+        node.parentNode.removeChild(node);
+    }
 }
 function detach_between(before, after) {
     while (before.nextSibling && before.nextSibling !== after) {
